ui/src/cache/cache.ts

```diff
@@ -53,13 +54,15 @@ export function asyncCache<TKey, TVal>(): AsyncCache<TKey, TVal> {
         setChangeTime(changeTime);
         const toFetch = ids.filter((id) => cache.get(cacheKey(id)) === null);
         if (toFetch.length > 0) {
-            const getting = getter(ids);
-            for (const id of toFetch) {
-                void put(
-                    cacheKey(id),
-                    getting.then((got) => got(id)),
-                );
-            }
+            chunk(ids, 50).forEach((chunk) => {
```

Comment:
Pitäisikö tässä chunkata `toFetch`:iä `ids`:in sijaan? Jos luen tätä oikein, niin nyt haetaan uudestaan myös jo kakusta löytyneet arvot

Reply:
Yes